source/connect/mongoclient.txt

```diff
@@ -0,0 +1,210 @@
+.. _kotlin-sync-mongoclient:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and perform 
+  opertions on the MongoDB deployment.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoClient`` object
+to connect to MongoDB.
+
+.. _kotlin-sync-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. Any ref for authentication? Are we planning on making a page for authentication?
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+       For more information about the ``authSource`` connection option, see
+       :ref:`pymongo-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`kotlin-sync-connection-options` for a full description of
+       these options.
+
+.. Is there relevant connection string content for kotlin sync to ref to? Only see for java sync.
```

Comment:
I would confirm with the Java DBX team but it's usually the case that Java and Kotlin content overlaps heavily.

When in doubt, we can always link readers to the [Server manual](https://www.mongodb.com/docs/manual/reference/connection-string/) for more general-purpose reading on a subject.

```suggestion
```

Reply:
Hi @jyemin, do you know if the connection string information for java sync is applicable for kotlin sync? I would like to link to connection string information on this page. This is the connection string page for java sync: https://www.mongodb.com/docs/manual/reference/connection-string/?tck=docs_driver_python

Reply:
Yes, connection string info is exactly the same.